website/versioned_docs/version-0.19.0/tutorial/index.mdx

```diff
@@ -182,7 +182,7 @@ html! {
 
 Refresh the browser page, and you should see the following output displayed:
 
-**TODO: Add screenshot**
+![Screenshot](https://user-images.githubusercontent.com/56165400/151808404-567c4d11-0898-46b6-9492-5ae8a11260b3.png)
```

Comment:
You should put your image as a file and commit it in `website/static` and use that instead.

Use `router-deserialization-failure-behavior.gif` as a reference for how to do this

Reply:
One question. What is the default URL for accessing the static files?

Reply:
@michaelgrigoryan25 same way as `router-deserialization-failure-behavior.gif`, i think it was just `/img/screenshot.png`

Reply:
Thank you @voidpumpkin.